toolchain/sem_ir/file.h

```diff
@@ -305,16 +312,23 @@ class File : public Printable<File> {
   auto filename() const -> llvm::StringRef { return filename_; }
 
  private:
-  // Allocates a copy of the given data using our slab allocator.
+  // Allocates an uninitialized array using our allocator.
   template <typename T>
-  auto AllocateCopy(llvm::ArrayRef<T> data) -> llvm::MutableArrayRef<T> {
+  auto AllocateUninitialized(std::size_t size) -> llvm::MutableArrayRef<T> {
     // We're not going to run a destructor, so ensure that's OK.
     static_assert(std::is_trivially_destructible_v<T>);
 
-    T* storage = static_cast<T*>(
-        allocator_.Allocate(data.size() * sizeof(T), alignof(T)));
-    std::uninitialized_copy(data.begin(), data.end(), storage);
-    return llvm::MutableArrayRef<T>(storage, data.size());
+    T* storage =
+        static_cast<T*>(allocator_.Allocate(size * sizeof(T), alignof(T)));
+    return llvm::MutableArrayRef<T>(storage, size);
+  }
+
+  // Allocates a copy of the given data using our slab allocator.
```

Comment:
```suggestion
  // Allocates a copy of the given data using our allocator.
```

(or add "slab" back to the comment above)

Reply:
Done.